integrations/ad_integration/ad_writer.py

```diff
@@ -65,6 +65,9 @@ def _other_attributes(self, mo_values, new_user=False):
             (write_settings['org_field'], mo_values['location'].replace('&', 'og'))
         ]
 
+        # Add SAM to mo_values
+        mo_values['name_sam'] = '{} - {}'.format(mo_values['full_name'], user_sam)
```

Comment:
Lagrer man for eksempel "Peter Nissen - pnissen" i den attribut?

Reply:
Ja